components/ui-patterns/Skeleton.tsx

```diff
@@ -0,0 +1,17 @@
+import Skeleton from 'react-loading-skeleton'
+import 'react-loading-skeleton/dist/skeleton.css'
+import { CardSkeletonType } from '../../types/ui-pattern-types/skeleton-types';
+
+function CardSkeleton({ visibility }: CardSkeletonType) {
+    return (
+        <Skeleton 
+            width="280px"
+            height="320px"
+            style={{
+                opacity: visibility
+            }}
```

Comment:
Hey @yashsehgal 👋

Do we have other props for the `Skeleton Wrapper Component` which are provided by the imported library
if yes, then

```
<Skeleton 
    width="280px"
    height="320px"
    style={{
      opacity: visibility
    }}
    {...props}
/>
```

Reply:
Yes, this is a nice add-on to this component, if you want you can add this in the `type`. ✅

Reply:
Sure @ayushsoni1010 👍🏽 I am currently working on this change only.

> Let's build the skeleton loading based on its data, like if we have images, headings, and descriptions, etc. So we have to make the UI similar to these components.

For this, I will making component wrappers, according to different use-cases.
Just like this `<EventCardSkeleton />`, `<TeamCardSkeleton />` and etc